### DISCUSSION TITLE
How to get extra space around the margin of a document having PDF Background

### DISCUSSION THREAD

**Discussion by MiltonBalaOfficial** [xournalpp/xournalpp#6043] [Q&A] [answered]
Hi, very often I need space around the margin of a document having PDF background. I have tried to make a plugin for increasing page Width and height. The size increases but the PDF background remains at the top-left corner and so I dont get space at top and left. Is it possible to set the PDF Background at centre? Or any other solution to solve my problem?

**Comment by rolandlo** [comment, marked as answer]
No, it's not possible to set the PDF background at the center. You could add space to the PDF itself e.g. by embedding the pages in a LaTeX document.

**Comment by PillowTrauma** [comment]
If you don't mind doing the margins first, you could try pdfCropMargins. 
https://pypi.org/project/pdfCropMargins/

**Reply by MiltonBalaOfficial** [reply]
Thanks for the suggestion.  I will give a try. But this plugin [[IncreaseAndDecreasePageDimensions](https://github.com/MiltonBalaOfficial/IncreaseAndDecreasePageDimensions)] serves me a lot.
